rules/manager.go

```diff
@@ -818,17 +824,28 @@ func (m *Manager) AlertingRules() []*AlertingRule {
 
 // Describe implements prometheus.Collector.
 func (m *Manager) Describe(ch chan<- *prometheus.Desc) {
+	ch <- lastEvaluation
 	ch <- lastDuration
 	ch <- groupInterval
 }
 
 // Collect implements prometheus.Collector.
 func (m *Manager) Collect(ch chan<- prometheus.Metric) {
 	for _, g := range m.RuleGroups() {
+		lastEvaluationTime := g.GetEvaluationTimestamp()
+		lastEvaluationTimestamp := 0.0
+		if !lastEvaluationTime.IsZero() {
+			lastEvaluationTimestamp = float64(timestamp.FromTime(lastEvaluationTime)) / 1e3
```

Comment:
Not sure why the indirection through `timestamp.FromTime()` is needed here. Couldn't you just do:

```suggestion
			lastEvaluationTimestamp = lastEvaluationTime.Seconds()
```

Reply:
The durations of evaluations are in milliseconds.  So `lastEvaluationTimestamp` keeps three decimal fractions. WDYT?

Reply:
The result of `lastEvaluationTime` is type of `time.Time`. Its `Seoncds()` only returns time in seconds. I think it should be changed to ` float64(lastEvaluationTime.Second()) + float64(lastEvaluationTime.Nanosecond())/1e9`. Just likes what `time.Duration.Seconds()` does. WDYT?

Reply:
My fault. `lastEvaluationTime.Second()` is not correct.

But `lastEvaluationTime` has no method named `Seconds`.
This is correct:
```go
lastEvaluationTimestamp = float64(lastEvaluationTime.UnixNano()) / 1e9
```

Reply:
Ah hurrdurr, I was thinking of the `time.Duration` type, which does have the `Seconds()` method. Yep, your suggestion makes total sense.